File: SecondYear/ksh1710_KshitizAgarwal_2226cse1033_2/Week_1/pattern9.cpp
void printTriangle(int n) {
        
	     for (int i = 1; i <= n; i++){
	         
             for (int j =n; j >= i; j--){
                   cout<<"*";
               }
             for (int k =2; k <= i; k++){
                  cout<<"  ";
               }
             for (int l =n; l >= i; l--){
                   cout<<"*";
               }
            cout << endl;
         }
	     for (int i = 1; i <= n; i++){
	         
             for (int j =1; j <= i; j++){
                   cout<<"*";
               }
             for (int k =n-1; k >= i; k--){
                  cout<<"  ";
               }
             for (int l =1; l <= i; l++){
                   cout<<"*";
               }
            cout << endl;
         }
    }

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_1/Pattern_9.py
#User function Template for python3

class Solution:
    def printTriangle(self, N):
        # Code here
        for i in range(1, N + 1):
            ab = "*" * (N - i + 1)
            sp = " " * (2 * (i - 1))
            print(ab + sp + ab)

        for i in range(N, 0, -1):
            ab = "*" * (N - i + 1)
            sp = " " * (2 * (i - 1))
            print(ab + sp + ab)


#{ 
 # Driver Code Starts
#Initial Template for Python 3

if __name__ == '__main__':
    t = int(input())
    for _ in range(t):
        N = int(input().strip())
        ob = Solution()
        ob.printTriangle(N)
# } Driver Code Ends